packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -293,6 +327,7 @@ export function patchFetch({
         if (typeof curRevalidate === 'number' || curRevalidate === false) {
           revalidate = curRevalidate
         }
+        validateRevalidate(curRevalidate, staticGenerationStore.urlPathname)
```

Comment:
curious why we need to validate this in multiple spots instead of when we initially call `let curRevalidate = getNextField('revalidate')` -- that's the only time it'd be "untrusted" right?

Reply:
Added as a sanity check to ensure an invalid value doesn't leak through